### PR TITLE
Made some changes on the Error 404 page. 

### PR DESCRIPTION
**Pull request by BillNas**

### CODE REVIEW

**Review by EmanuelGF — CHANGES_REQUESTED** [review]
Hi @BillNas l! 🙏
First off all let me thank you for your interest and help on the project! 👌
The 404 page looks much better!  Just a couple of things to refactor out:
1- Can you fix the spacing? 


2- On **src/App.js** the import path to **notFound** should be corrected. 

 Note: your PR includes a lot of changes to the package-lock file from your local npm install run. you can avoid this next time by stashing the package-lock changes after npm install then  doing a "pull" of any changes from the "main" branch of the "upstream" into your local repository.
Thank you again for your time spent on this.

**Comment by BillNas** [comment]
I fixed the spacing. Hope it is good now. 🙏🏻

**Review by EmanuelGF — CHANGES_REQUESTED** [review]
Hey @BillNas  ! Hope you are doing good ✌
After having a better look at the code you summitted I spotted some other issues that I have commented on. 
Furthermore there is another issue. It looks like when committed your changes you did not sync your repo getting the latest changes and updates from the original repo. If I were to merge your pull request it would roll back the history removing some latest changes. 
It's good practice to always pull latest changes and merge before submitting your commits. If you can try to fix this it would be gold 🙂
Let me know how its going.

File: src/Components/NotFound/notFound.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.error {
+	padding: 0;
+	margin: 0;
```
[the remark this answers is not in the document]
Comment (BillNas) [reply src/Components/NotFound/notFound.scss:3]: I am a beginner so I don't exactly know how to handle that kind of situations like sync of a repo. 😣

**Comment by EmanuelGF** [comment]
No worries ! 🙂
I will not merge this PR but I have already accepted your code changes and given attribution to you! 👌 Thank you again for your effort and help! NICE!!
If you want to keep exploring this project don't forget to pull the newest added changes from the upstream and pull them to your main on your local machine.
I suggest you go through all the guides on the CONTRIBUTION file to get a hang on git.  
Star and watch this project to get notified on more issues like this one! 
Thanks again.

**Pull request closed without merging**